journals/apps/journals/models.py

```diff
@@ -625,6 +626,26 @@ def get_user_pages(user, pages=None):
 
         return user_pages
 
+    @staticmethod
+    def get_user_collections(user, collections=None):
+        """
+        Args:
+            user: instance of User model
+            collections: queryset of collections to filter
+        Returns: wagtail collections queryset where given user has add, change permissions
+        if collections queryset is provided filter is applied on that.
+        """
+        if not collections:
+            collections = Collection.objects.all()
+
+        collection_permission_policy = CollectionOwnershipPermissionPolicy(
+            Document, owner_field_name='uploaded_by_user'
```

Comment:
Can you explain what this is doing? It's confusing that it's passing Document (would have thought Image or Document) and what is significance of owner_field_name='uploaded_by_user'?

Reply:
@bill-filler `CollectionOwnershipPermissionPolicy` can only be instantiated by passing two args one is the model(Document or Image) which a collection belongs to and other is the name of the field which can be use to determine owner of the entry in that model. I had to pass these args to just instantiate `CollectionOwnershipPermissionPolicy`.  These args don't have any impact on the outcome of this method.